src/util/app.js

```diff
@@ -27,6 +27,15 @@ export const appQueries = {
             key: SYSTEM_SETTINGS,
         },
     },
+    systemInfo: {
+        resource: 'system/info',
```

Comment:
Perhaps only request `calendar` to avoid a "bloated" response with information we don't use.

Reply:
Addressed here: [a594f89](https://github.com/dhis2/maps-app/pull/3269/commits/a594f8963087a3dbdcb0eecdc003027de47558e7)